### DISCUSSION TITLE
Address for testnet contract?

### DISCUSSION THREAD

**Discussion by empeje** [f8n/fnd-protocol#60] [Q&A] [answered]
I'm currently integrating foundation to my app and need to test the interaction in the test contract? Which one is the recommended process? Is using test net is the recommended one, can I have the address of the contract in the test net?

If running everything locally is the recommended step, is there any guide to set it up completely in local?

**Comment by HardlyDifficult** [comment, marked as answer]
We use the Goerli network for testing. The addresses are under staging, network 5 here https://github.com/f8n/fnd-protocol/blob/main/addresses.js

The market contract on Goerli is at 0xeB1bD095061bbDb1aD065524628812cae63e4222

Let me know if there's anything else you need.

**Comment by empeje** [comment]
This is really useful, thank you @NickCuso
